src/app/Signup/page.tsx

```diff
@@ -0,0 +1,182 @@
+    e.preventDefault();
+
+    EsetError('');
+    PsetError('');
+    confirmPsetError('');
+    setShakeEmail(false);
+    setShakePassword(false);
+    setShakeConfirmPassword(false);
+
+    let valid = true;
+
+    // Validate email
+    if (!email) {
+      EsetError('Email cannot be empty.');
+      setShakeEmail(true);
+      valid = false;
+    } else if (!emailRegex.test(email)) {
+      EsetError('Invalid email address.');
+      setShakeEmail(true);
+      valid = false;
+    }
+
+    // Validate password
+    if (!password) {
+      PsetError('Password cannot be empty.');
+      setShakePassword(true);
+      valid = false;
+    }
+
+    // Validate confirm password
+    if (!confirmpassword) {
+      confirmPsetError('Please confirm password.');
+      setShakeConfirmPassword(true);
+      valid = false;
+    } else if (password !== confirmpassword) {
+      confirmPsetError('Passwords does not match.');
+      setShakeConfirmPassword(true);
+      valid = false;
+    }
+
+    if (!valid) {
+      setTimeout(() => {
+        setShakeEmail(false);
+        setShakePassword(false);
+        setShakeConfirmPassword(false);
+      }, 100);
+      return;
+    }
+  };
+
+  return (
+    <main className={styles.SignUpPageContainer}>
+      <div className={styles.headerconatiner}>
+        <h1 className={styles.CreateAccount}>Create Account</h1>
+      </div>
+      {/* Email and Password Input Fields */}
+      <form onSubmit={handleSubmit} noValidate>
+        <div className={styles.EmailContainer}>
+          <label className={styles.TextBoxLabel}>Email</label>
+          <InputField
+            type="email"
+            value={email}
+            onChange={handleEmailChange}
+            placeholder="Enter email"
+            label="Password"
+            errorMessage={emailError}
+            shake={shakeEmail}
+          />
+        </div>
+
+        <div className={styles.PasswordContainer}>
+          <label className={styles.TextBoxLabel}>Password</label>
+          <InputField
+            type="password"
+            value={password}
+            onChange={handlePasswordChange}
+            placeholder="Enter password"
+            label="Password"
+            errorMessage={passError}
+            shake={shakePassword}
+          />
+        </div>
+
+        <div className={styles.ConfirmPasswordContainer}>
+          <label className={styles.TextBoxLabel}>Confirm Password</label>
+          <InputField
+            type="password"
+            value={confirmpassword}
+            onChange={handleConfirmPasswordChange}
+            placeholder="confirm password"
+            label="Password"
+            errorMessage={ConfirmpassError}
+            shake={shakeConfirmPass}
+          />
+        </div>
+
+        {/* Buttons for Login and Signup */}
+        <div className={styles.signbttncontainer}>
+          <InactiveButton text="Sign up" />
+        </div>
+        <span className={styles.span}>or</span>
```

Comment:
the `or` font size is too small and needs to be increased, apply this change to other changes as well please